.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Do we want to track this here? This can be fluid over time.

Reply:
Assignees most likely not. I think title and labels can be optional and could provide additional information. As we are contributors, we can add labels directly so maybe that is not as useful to us. I will remove assignees.